### ISSUE TITLE
04_error

### ISSUE THREAD

**Issue by KellyBlack** [mandli/intro-numerical-methods#63]
In the 04_error.ipynb notebook the two categories of numerical errors are listed as truncation error and floating point error. I tend to think of truncation error as the error made in a single step of approximating an ODE. I would like to separate truncation error into two other different kinds of errors, discretization error (error associated with using a simpler function) and convergence error (errors that accumulate over multiple steps in an algorithm). 

Would this be okay?

**Comment by mandli** [comment]
I agree, this is often also a point of confusion for students when they see FD (convergence of one point's derivative vs. a set of points) and the simple proof of stability for forward Euler's method.  For the latter we call it "global error", is that what you are thinking of?

**Comment by KellyBlack** [comment]
Yes, this is what I was thinking of. In the context of the 04_error notebook, if I change it to discuss error in terms of model error, discretization error, accumulation error (convergence error would be a bad term), and floating point error would that be acceptable?  I would add an example of accumulation error using a difference equation that is not stable.

**Comment by mandli** [comment]
That sounds good to me.
